code/game/gamemodes/dynamic/dynamic.dm

```diff
@@ -291,9 +301,11 @@ GLOBAL_VAR_INIT(dynamic_forced_threat_level, -1)
 			. += G.get_report()
 
 	print_command_report(., "Central Command Status Summary", announce=FALSE)
-	priority_announce("A summary has been copied and printed to all communications consoles.", "Enemy communication intercepted. Security level elevated.", "intercept")
-	if(GLOB.security_level < SEC_LEVEL_BLUE)
-		set_security_level(SEC_LEVEL_BLUE)
```

Comment:
The new send_intercept no longer sets the security level at all. The roundstart security level doesn't get set at all with this PR. `raise_security_level()` and `lower_security_level()` are defined in this PR, but they aren't actually used anywhere.

Reply:
this was a bug from a completely different PR, fixed in #15813